internal/plugins/openshift/v1/init.go

```diff
@@ -65,7 +65,7 @@ var imageSubstitutions = map[string][]substitution{
 	filepath.Join("config", "default", "manager_auth_proxy_patch.yaml"): {
 		{
 			regexp.MustCompile(`gcr.io/kubebuilder/kube-rbac-proxy:[^ \n]+`),
-			"registry.redhat.io/openshift4/ose-kube-rbac-proxy:v" + ocpProductVersion,
+			"registry.redhat.io/openshift4/ose-kube-rbac-proxy-rhel9:v" + ocpProductVersion,
```

Comment:
[Non-blocking review] It'd be better to adapt a uniform way of image naming convention [component-name]+[rhel version] or [ubi version]+[ocp product version] or [ubi minimal version]. Example strings would be like: ose-ansible-operator-rhel9-4.18 or ose-ansible-operator-ubi9-9.2

Reply:
The image tag is generated by the ART team. Hi @joepvd Please help to take a look. Can probably provide that context here.Thanks

Reply:
This would need changes in the release configurations, it is ok to look at it outside the context of this PR.

Reply:
We need to check with ART if that's possible or not. AFAIK there are certain naming conventions and requirements for images, and there are some dependent components as well on this image : https://github.com/openshift-eng/ocp-build-data/blob/openshift-4.17/images/kube-rbac-proxy.yml#L13-L24